Rules/TDM/gamemode.cfg

```diff
@@ -24,7 +24,7 @@ scripts                                           = KAG.as;
 													TimeToEnd.as;
 													RestartAfterShortPostGame.as;
 													FanfareOnWin.as;
-													TeamWinBanner.as;
+													GameStateBanners.as;
```

Comment:
Other gamemodes shouldn't include this as it is a dependency script

Reply:
resolved